guide/case-studies/payjoin.md

```diff
@@ -109,7 +109,7 @@ In the payjoin process, parties edit, sign and pass iterations of the transactio
 
 ### Background
 
-The payjoin case study has its origins in a [design challenge](https://bitcoin.design/guide/resources/design-challenges/#challenge-6-private-purchase), which entailed creating a sender flow for payment using a Payjoin transaction.
+The payjoin case study has its origins in a [design challenge]({{ '/guide/resources/design-challenges/#challenge-6-private-purchase' | relative_url }}), which entailed creating a sender flow for payment using a Payjoin transaction.
 
 While payjoin provides privacy improvements and potential fee-savings, they present the challenge of coordination between transacting parties and may require new user flows to implement features enabled through [BIP-78](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki).
```

Comment:
```suggestion
While payjoin provides privacy improvements and potential fee-savings, they present  unique challenges. One of these is the need of coordination between transacting parties to implement features through [BIP-78](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki). This would require modifications to the userflow. 
```